client/src/App.js

```diff
@@ -1,32 +1,28 @@
 import { Route, Routes } from "react-router-dom";
 import SignupForm from "./Components/forms/SignupForm";
 import LoginForm from "./Components/forms/LoginForm";
-import Home from "./Components/layout/HomePage/Home";
 
-// import { Footer } from "./Components/layout/footer/Footer";
 import Footer from "./Components/layout/footer/Footer";
 import LogOut from "./Components/layout/LogOut/LogOut";
-
-
-
+import ClockIn from "./Components/layout/ClockIn/ClockIn";
+import VolunteerClockIn from "./Components/layout/ClockIn/VolunteerClockIn";
+import "./app.css";
+import Header  from "./Components/layout/header/Header";
 
 const App = () => (
-	<div>
-		<Home />
-
-		<Routes>
-			<Route exact path="/" element={<LoginForm />} />
-
-
-			<Route
-				path="/signup"
-				element={<SignupForm />}
-			/>
-{/* /signup */}
-			<Route path="/SignupForm/this/site" element={<SignupForm />} />
-			<Route path="/" element={<LogOut />} />
-		</Routes>
-	 <Footer />
+	<div className="main-wrapper">
+		<Header />
+		<div className="main-container">
+			<Routes>
+				<Route path="/" element={<LoginForm />} />
+				<Route path="/signup" element={<SignupForm />} />
+
+				<Route path="/" element={<LogOut />} />
+				<Route path="/" element={<ClockIn />} />
+				<Route path="/clockin" element={<VolunteerClockIn />} />
+			</Routes>
```

Comment:
Can the <logOut />'s path name be part of both trainee and volunteer clock in page? At the moment logout buttons is showing on sign up page as well. so perhaps, <logOut/> path could be path="/clockin", trainee clockin path="/clockin/trainee" and volunteer clockin path="/clockin/volunteer".